tests/administration/users.test.js

```diff
@@ -3,54 +3,64 @@ const common = require('../common'),
       expect = require('chai').expect;
 
 describe('Users', () => {
-
+  
   beforeEach(() => {
     return common.login(browser, 'admin', 'admin', 'local' )
       .then(() => common.clickSidebarTab(browser, 'Users', 'System Users'))
   });
 
-  //it('should navigate to Users', () => common.clickSidebarTab(browser, 'Users', 'System Users'));
+  afterEach(() => common.logout());
   
   it('should allow password changes for admin', () => {
-  	return browser.click('//td[contains(text(),"admin") and position()=1]/..//button[contains(@id,"cp")]')
-  	  .then(() => browser.setValue('//td[contains(text(),"admin") and position()=1]/..//form//input[@id="password"]', 'admin'))
-  	  .then(() => browser.click('//td[contains(text(),"admin") and position()=1]/..//form//button[@type="submit"]'))
+  	return browser.waitUntil(
+        browser.getAttribute('//td[text()="admin" and position()=1]/..//button[contains(@id,"cp")]', 'onclick')
+          .then((cl) => { return cl !== null; })
+      ,10000)
+      .then(() => browser.click('//td[text()="admin" and position()=1]/..//button[contains(@id,"cp")]'))
+  	  .then(() => browser.waitForExist('//td[text()="admin" and position()=1]/..//form//input[@id="password"]'))
+      .then(() => browser.setValue('//td[text()="admin" and position()=1]/..//form//input[@id="password"]', 'admin'))
+      .then(() => browser.waitUntil(
+        browser.getAttribute('//td[text()="admin" and position()=1]/..//form//button[@type="submit"]', 'type')
+          .then((cl) => { return cl !== null; })
+      ,10000))
+  	  .then(() => browser.click('//td[text()="admin" and position()=1]/..//form//button[@type="submit"]'))
       .then(() => browser.waitForExist('//h1[@class="page-header"]'))
       .then(() => browser.getText('//h1[@class="page-header"]'))
   	  .then((text) => expect(text).to.equal('System Users'));
   });
```

Comment:
Change password test should probably be more complete, and include, changing it to something that isn't the same default password -> logout -> validate the password was set correctly, and then change it back to the default